packages/sync/src/conflicts/conflictLink.ts

```diff
@@ -24,7 +24,7 @@ export const conflictLink = (config: DataSyncConfig): ApolloLink => {
 
   return onError(({ response, operation, forward, graphQLErrors }) => {
     const data = getConflictData(graphQLErrors);
-    if (data && config.conflictStrategy && config.conflictStateProvider) {
+    if (data && (config.conflictStrategy || config.resolvers) && config.conflictStateProvider) {
```

Comment:
IMHO having two ways to handle conflicts is really confusing and also introduces some complexity. We do not need to introduce another config - just support alternative way of configuration on the same config key.